python/ct/serialization/tls_message_test.py

```diff
@@ -209,6 +222,12 @@ def test_encode_repeated_too_short_fails(self):
         test_message.ClearField("vector_uint32")
         self.verify_encode_fails(test_message)
 
+    def test_encode_repeated_fixed_length_too_short_fails(self):
```

Comment:
I think you should also test that the encoder rejects components that aren't exactly `fixed_bytes` long.

And also test `test_encode_repeated_fixed_length_too_long_fails`